selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -54,6 +56,16 @@ WifiManager::WifiManager(){
     raw_adapter_state = get_response<uint>(response);
     change(raw_adapter_state, 0, 0);
   }
+  // Compute tethering ssid as "Weedle" + first 4 characters of serial number of a device
+  tethering_ssid = "Weedle";
+  QString path = "DongleId";
+  std::vector<char> bytes = Params().read_db_bytes(path.toStdString().c_str());
```

Comment:
Just hardcode as `Params().read_db_bytes("DongleId");`

Reply:
also use get, we're trying to move everything to the python params API